_sass/base.scss

```diff
@@ -1721,15 +1721,17 @@ em {
   color: $blue-dark;
   box-shadow: none;
   flex: 1;
-  background-image: url(../images/icon-search.svg);
```

Comment:
Try retaining this block and play around a little with opacity or jquery attribute fetch